proxy/http2/Http2ConnectionState.cc

```diff
@@ -1120,6 +1120,9 @@ Http2ConnectionState::release_stream(Http2Stream *stream)
     // We were shutting down, go ahead and terminate the session
     ua_session->destroy();
     ua_session = nullptr;
+  } else if (total_client_streams_count == 0 && http2_drain && ua_session && stream) {
+    send_goaway_frame(stream->get_id(), Http2ErrorCode::HTTP2_ERROR_NO_ERROR);
```

Comment:
Is this the GOAWAY frame to notify Graceful Shutdown to client? If so the stream id should be 2^31 - 1
RFC says below in [section 6.8](http://httpwg.org/specs/rfc7540.html#GOAWAY).
> A server that is attempting to gracefully shut down a connection SHOULD send an initial GOAWAY frame with the last stream identifier set to 231-1 and a NO_ERROR code.